server/index.js

```diff
@@ -69,7 +69,7 @@ async function startServer() {
 
   addWebhookHandlers();
 
-  app.get('/auth', async (req, res) => {
+  app.get('/login', async (req, res) => {
```

Comment:
Since the older version of the CLI uses `/auth`, should we update the docs to specify you need to be on the latest version of the CLI?

Reply:
good call